dags/cluster/tasks/airflow_logic/cluster_acteurs_clusters_display_task.py

```diff
@@ -0,0 +1,147 @@
+import logging
+
+import numpy as np
+import pandas as pd
+from airflow import DAG
+from airflow.exceptions import AirflowSkipException
+from airflow.operators.python import PythonOperator
+from cluster.config.model import ClusterConfig
+from cluster.tasks.airflow_logic.task_ids import (
+    TASK_CLUSTERS_DISPLAY,
+    TASK_CONFIG_CREATE,
+    TASK_NORMALIZE,
+)
+from cluster.tasks.business_logic import (
+    cluster_acteurs_add_original_df_columns,
+    cluster_acteurs_clusters,
+    cluster_acteurs_df_sort,
+    cluster_acteurs_selection_children,
+)
+from utils import logging_utils as log
+from utils.django import django_setup_full
+
+django_setup_full()
+
+from data.models.change import (  # noqa: E402
+    COL_ENTITY_TYPE,
+    ENTITY_ACTEUR_DISPLAYED,
+    ENTITY_ACTEUR_REVISION,
+)
+from qfdmo.models import RevisionActeur  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+
+def task_info_get():
+    return f"""
+
```

Comment:
```suggestion
logger = logging.getLogger(__name__)


def task_info_get():
    from data.models.change import (
        COL_ENTITY_TYPE,
        ENTITY_ACTEUR_DISPLAYED,
        ENTITY_ACTEUR_REVISION,
    )
    from qfdmo.models import RevisionActeur


    return f"""

```

est-ce que les imports n'auraient pas intérêt à être fait directement dans le corps de la fonction ?
Ça évite le noqa